runtime/codert_vm/arm64nathelp.m4

```diff
@@ -126,7 +126,11 @@ define({OLD_DUAL_MODE_HELPER},{
 	CALL_C_WITH_VMTHREAD(old_fast_$1)
 	cbz x0,.L_done_$1
 	SAVE_C_NONVOLATILE_REGS
-	hlt {#}0
+	mov x1,x0
+	mov x0,J9VMTHREAD
+	blr x1
+	cbz x0,+8
```

Comment:
Rather than hard-coding the branch displacement `+8`, would it be clearer to branch to an actual named label?  It makes the code easier to follow.  Unless you have another reason for hard-coding this?  This comment applies to the other `cbz` instructions in the other macros as well.

Reply:
Added labels based on those in `pnathelp.m4`.